docs/tutorials/09_Protein_Folding.ipynb

```diff
@@ -749,6 +827,206 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig = result_2.get_figure(title=\"Protein Structure\", ticks=True, grid=True)\n",
+    "fig.show()"
```

Comment:
There are different shades because they try to imitate a depth effect. It is done automatically by the 3D plotting tools and hence not reflected in the legend.

Reply:
Ah ok, I figured there had to be some reasonable explanation for the color difference. In the plot function we have there is no option to turn that off - I see the default for 3D plot is on. May that would be useful? A user can of course plot the data anyway they like since they have access to it - they do not need to use the built in figure capability provisioned here.

Reply:
I am not sure I understand. The plot needs to be "3D" even if it doesn't rotate since it represents the shape of the protein. How would we turn that off?